docs/user_guide/source/advanced/ecp_hardware.rst

```diff
@@ -33,27 +33,29 @@ your tests.
 Examples of launching ADIOS2 SST unit tests using MPI DP:
 
 .. code-block:: bash
- # We omit some of the srun (SLURM) arguments which are specific of the project
- # you are working on. Note that you could avoid calling srun directly by
- # setting the CMAKE variable `MPIEXEC_EXECUTABLE`.
 
- # Launch simple writer test instance
- srun {PROJFLAGS }-N 1 /gpfs/alpine/proj-shared/csc331/vbolea/ADIOS2-build/bin/TestCommonWrite SST mpi_dp_test CPCommPattern=Min,MarshalMethod=BP5'
+  # We omit some of the srun (SLURM) arguments which are specific of the project
+  # you are working on. Note that you could avoid calling srun directly by
+  # setting the CMAKE variable `MPIEXEC_EXECUTABLE`.
 
- # On another terminal launch multiple instances of the Reader test
- srun {PROJFLAGS} -N 2 /gpfs/alpine/proj-shared/csc331/vbolea/ADIOS2-build/bin/TestCommonRead SST mpi_dp_test
+  # Launch simple writer test instance
+  srun {PROJFLAGS }-N 1 /gpfs/alpine/proj-shared/csc331/vbolea/ADIOS2-build/bin/TestCommonWrite SST mpi_dp_test CPCommPattern=Min,MarshalMethod=BP5
```

Comment:
```suggestion
  srun {PROJFLAGS } -N 1 /gpfs/alpine/proj-shared/csc331/vbolea/ADIOS2-build/bin/TestCommonWrite SST mpi_dp_test CPCommPattern=Min,MarshalMethod=BP5
```

Not a change in this PR but good to change :)

Reply:
Fixed